rpm/centos7/scripts/after_install.sh

```diff
@@ -65,9 +66,10 @@ if [ ! -f "$APP_LOG_CONFIG" ]; then
 fi
 
 # Generate plugin logging config if it doesn't exist
-if [ ! -f "$PLUGIN_LOG_CONFIG" ]; then
+if [ ! -f "$REMOTE_PLUGIN_LOG_CONFIG" ] || [ ! -f "$LOCAL_PLUGIN_LOG_CONFIG" ]; then
```

Comment:
I wouldn't expect the remote and local plugin log config files that get generated to be any different and looking at the changes to `config.py` it appears that they are in fact the same.  That makes me think that the change to have separate `local_conf` and `remote_conf` parameters isn't really what you want.  Instead, I think it makes sense to just run this command twice, once with `--config-file "$LOCAL_PLUGIN_LOG_CONFIG"` and then again with `--config-file "$REMOTE_PLUGIN_LOG_CONFIG"`.

This becomes moot if you do the above, but I think as written this would overwrite a present config file if the other one happened to be missing, which seems like not the desired result.

Reply:
This script is only run on installation, so I can't see a situation this would be run and the script overwrites a file. Regardless, I do agree that it's weird to do it this way. Fixed in latest.